src/wizards/services/WizardService.ts

```diff
@@ -92,7 +92,7 @@ export class WizardService {
     const wizardState = this.getWizardState(wizardManager);
     const indexOfActive = wizardState.indexOfActive;
 
-    this.goToStage(wizardManager, indexOfActive + 1, true);
+    await this.goToStage(wizardManager, indexOfActive + 1, true);
```

Comment:
remove `await` and `async`.  They aren't needed and cause confusion.

Reply:
Actually, they are because the proceed button uses it to show the loading icon. If we don't show the loading icon, the user won't know the Proceed button is doing something. The loading was added because Stage 5 takes about 2-3 sec to validate

Reply:
I don't see how this is correct.  The caller is responsible for doing the await.